<launch>

<!-- transformation between robots -->
<node pkg="tf" type="static_transform_publisher" name="robot2_to_robot1" args="0 3.8 0 0 0 0 robot1/map robot2/map 20" /> 

<node pkg="mapmerge" type="mapmerge_node" name="map_merge" output="screen" >
	<param name="robot1_x" value="0.0" />
	<param name="robot2_x" value="0.0" />
	<!-- <param name="robot3_x" value="0.0" /> -->
	<param name="robot1_y" value="0.0" />
	<param name="robot2_y" value="3.8" />
	<param name="resolution" value="0.05" />
	<param name="map_width"  value="600"  />
	<param name="map_height" value="600"  />
</node>

  <node pkg="mapmerge" type="mapmerge_costmap_node" name="map_merge_costmap" output="screen" >
	<param name="robot1_x" value="0.0" />
	<param name="robot2_x" value="0.0" />
	<param name="robot1_y" value="0.0" />
	<param name="robot2_y" value="3.8" />
	<param name="resolution" value="0.05" />
	<param name="map_width" value="600" />
	<param name="map_height" value="600" />
	<param name="robot1costmap" value="/robot1/move_base/global_costmap/costmap" />
	<param name="robot2costmap" value="/robot2/move_base/global_costmap/costmap" />
	<param name="robot3costmap" value="/robot3/move_base/global_costmap/costmap" />
</node>
</launch>